code/script.js

```diff
@@ -1,7 +1,24 @@
 // All the DOM selectors stored as short variables
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+const playAgainButton = document.getElementById('playAgain')
+const startButton = document.getElementById('start')
+const filter = document.getElementById('filter')
+const startWindow = document.getElementById('start-window')
+
+const startAudio = document.getElementById('audio')
+const winningAudio = document.getElementById('audio-winning')
+const lostAudio = document.getElementById('audio-lost')
+
+const modal = document.getElementById("myModal");
```

Comment:
maybe use a more describing name of this variable

Reply:
I must be better to name classes, id and variables :) but sometimes it is hard. Thanks I will think of that in my future projects!